dashboard/src/components/DeploymentForm/DeploymentForm.tsx

```diff
@@ -152,6 +165,7 @@ class DeploymentForm extends React.Component<IDeploymentFormProps, IDeploymentFo
                   onChange={this.handleChartVersionChange}
                   value={this.props.chartVersion}
                   required={true}
+                  disabled={hr ? true : false}
```

Comment:
at the moment I have disabled the version selection while performing an upgrade. I need to figure out how to propagate the version change in the form

Reply:
update: version selection is now enabled